packages/gatsby/src/internal-plugins/query-runner/page-query-runner.js

```diff
@@ -90,8 +90,9 @@ const runQueriesForIds = ids => {
     return Promise.resolve()
   }
   const state = store.getState()
-  return Promise.all(
-    ids.map(id => {
+  return Promise.mapSeries(
```

Comment:
we should maybe try and not depend on bluebird's static methods as it makes switching to the native promise eventually harder